### PR TITLE
Fix tests post constrain adapter

### PR DESCRIPTION
**Pull request by djowett-ftw**
This is part of the solution to https://github.com/4teamwork/ftw.publisher.sender/issues/76

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -4,3 +4,6 @@ extensions = mr.developer
 
 auto-checkout =
   ftw.publisher.core
+
+[branches]
+ftw.publisher.core = dj_abbreviate_constrained_types_when_poss
```
Comment (busykoala) [inline sources.cfg]: Please remove this and wait for `ftw.publisher.core` was merged. I don't think there are circular dependencies.

File: ftw/publisher/sender/tests/assets/folder_dx.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "unicode:backreferences_adapter": {},
+    "unicode:constrain_types_adapter": {"utf8:mode": -1},
```
Comment (busykoala) [inline ftw/publisher/sender/tests/assets/folder_dx.json]: Same as in receiver I'm not quite sure what the constrain does here.

Comment (djowett-ftw) [reply ftw/publisher/sender/tests/assets/folder_dx.json]: It's a consequence of https://github.com/4teamwork/ftw.publisher.core/pull/66
this file is the expected results of test_dexterity_folder_extractor

File: sources.cfg (hunk 1 above)
Comment (djowett-ftw) [reply sources.cfg]: sure - will do, but I needed it for development and proving tests pass for review, right?

Comment (mbaechtold) [inline sources.cfg]: 🚧 This must not be merged.

Comment (djowett-ftw) [reply sources.cfg]: Sure - it's gone !  .... still :-)

**Pull request merged**